src/net_processing.cpp

```diff
@@ -2777,7 +2781,7 @@ void PeerManagerImpl::UpdatePeerStateForReceivedHeaders(CNode& pfrom, Peer& peer
 
     // If we're in IBD, we want outbound peers that will serve us a useful
     // chain. Disconnect peers that are on chains with insufficient work.
-    if (m_chainman.ActiveChainstate().IsInitialBlockDownload() && !may_have_more_headers) {
+    if (m_chainman.IsAnyChainInIBD() && !may_have_more_headers) {
```

Comment:
c7cdc7ca833ce1b5ce9a696fb4ef633c55528023: this line does more than "add commentary". Maybe move this to 3a7a714721a593e271ce70d4612c995968dbb0da or its own commit directly after that one?

The code comment could clarify why we care about this even for the background sync: "with insufficient work" -> "with less than the minimum chain work."

Reply:
Fixed, thanks. Though I haven't updated the comment; having trouble figuring out what to say there.